region/lowernorfair/west.json

```diff
@@ -503,11 +545,41 @@
               "id": 1,
               "strats": [
                 {
-                  "name": "Base",
-                  "notable": false,
+                  "name": "GT Crumble Jump from Item",
+                  "notable": true,
                   "requires": [
                     "canNavigateHeatRooms",
+                    "canCrumbleJump",
                     {"heatFrames": 150}
+                  ],
+                  "note": "It takes a short hop to avoid hitting the ceiling to do this with only one crumble jump."
```

Comment:
a short hop and precise wall jump avoids the crumble blocks.

Reply:
I don't understand this one

Reply:
...Oh. Walljump off the far end. Wow.